docker/centos/Dockerfile

```diff
@@ -1,29 +1,32 @@
 FROM centos:latest
 WORKDIR /build
+ADD . /build/parity
```

Comment:
Add command should be moved right before `RUN cd parity` to utilized Docker cache for all deps and rust compilation.